app/jobs/tracking_sheet_report_job.rb

```diff
@@ -0,0 +1,138 @@
+# frozen_string_literal: true
+
+##
+# Job to create downloadable tracking sheets
+class TrackingSheetReportJob < GenericJob
+  ##
+  # A job that...
+  # @param [Integer] bulk_action_id GlobalID for a BulkAction object
+  # @param [hash] params additional parameters that an Argo job may need
+  # @option params [Array] :druids required list of druids
+  def perform(bulk_action_id, params)
+    super
+
+    druids = params[:druids]
+
+    with_bulk_action_log do |log|
+      update_druid_count
+      begin
```

Comment:
Is there a reason you aren't calling the existing `TrackSheet` code here?

Reply:
There was. I don't recall why. I'll rework to use the existing and see if it raises any issues.

Reply:
@jcoyne fixed. I had a mistaken assumption about how the files were written early so copied the code... That was unnecessary, so the existing `TrackSheet` model works fine. Good catch.